src/language-css/printer-postcss.js

```diff
@@ -924,12 +924,18 @@ function genericPrint(path, options, print) {
       }
 
       if (!node.open) {
+        const parentParentParentNode = path.getParentNode(2);
+        const forceHardLine =
+          parentParentParentNode.type === "css-decl" ||
+          (parentParentParentNode.type === "css-atrule" &&
+            parentParentParentNode.variable);
+
         const printed = path.map(print, "groups");
-        const res = [];
+        const res = forceHardLine ? [hardline] : [];
 
         for (let i = 0; i < printed.length; i++) {
           if (i !== 0) {
-            res.push([",", line]);
+            res.push([",", forceHardLine ? hardline : line]);
```

Comment:
how to fix this lint warning?

Reply:
@fisker can you please advise how to fix the lint error?

Reply:
This may work https://github.com/prettier/prettier/blob/310a5b1ba51ad6a3115670058d53a7ecb1dcd25c/src/language-js/printer-estree.js#L193

Reply:
Thank you 👍